src/components/ChickPeaWelcomePage.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
```

Comment:
@Eliasadam Similar to the three recipe slides component. You should have just one WelcomePage component that renders the correct recipeslides component depending on what the user chose in the Recipepreparation component. 👍